manifests/params.pp

```diff
@@ -1,27 +1,46 @@
 class dhcp::params {
 
   $dnsdomain = [$::domain]
+  $pxefilename = 'pxelinux.0'
 
   case $::osfamily {
     'Debian': {
       $dhcp_dir    = '/etc/dhcp'
       $packagename = 'isc-dhcp-server'
       $servicename = 'isc-dhcp-server'
       $root_group  = 'root'
+      $bootfiles   = {
+        '00:06' => 'grub2/bootia32.efi',
+        '00:07' => 'grub2/bootx64.efi',
+        '00:09' => 'grub2/bootx64.efi',
+      }
     }
 
     /^(FreeBSD|DragonFly)$/: {
       $dhcp_dir    = '/usr/local/etc'
       $packagename = 'isc-dhcp43-server'
       $servicename = 'isc-dhcpd'
       $root_group  = 'wheel'
+      $bootfiles   = {}
     }
 
     'RedHat': {
       $dhcp_dir    = '/etc/dhcp'
       $packagename = 'dhcp'
       $servicename = 'dhcpd'
       $root_group  = 'root'
+      if 0 + $::operatingsystemmajrelease > 6 {
```

Comment:
We still avoid using this fact because older facter versions don't support it, but I haven't seen this workaround before. Will puppet automatically convert the string to an int?